ipam/ring/ring_test.go

```diff
@@ -506,6 +506,22 @@ func TestOwner(t *testing.T) {
 
 }
 
+func makePeerName(i int) mesh.PeerName {
+	peer, _ := mesh.PeerNameFromString(fmt.Sprintf("%02d:%02d:00:00:00:00", i/10, i%10))
```

Comment:
Why 10 rather than 100?

Also, we have code like this in nameserver_test too. and in mesh.peers_test. and in ipam/testutils_test (would need some tweaking), and in TestFuzzRing.